packages/suite/src/components/suite/modals/CoinmarketReviewTransaction/components/Detail.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
```

Comment:
is this the same as `modals/ReviewTransaction/Details`? could it be reused?

Reply:
Coinmarket cross-referenced Detail.tsx from ReviewTransaction. As there were changes in Detail.tsx I decided to copy the previous version of it and move it to Coinmarket as its submodule. We can make it reusable for both usages in a next PR